src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -763,7 +763,7 @@ public void TestStartOnUnixWithBadPermissions()
             Assert.NotEqual(0, e.NativeErrorCode);
         }
 
-        [PlatformSpecific(TestPlatforms.AnyUnix)]
+        [PlatformSpecific(TestPlatforms.AnyUnix)]  // Uses P/Invokes to get permissions
```

Comment:
Nit: both this test and the one above it use a P/Invoke to set permissions, not get them